graphics/palette.h

```diff
@@ -111,9 +111,98 @@ class PaletteManager : Common::NonCopyable {
 
 namespace Graphics {
 
+struct Palette {
+	byte data[256 * 3];
+	uint size;
```

Comment:
Would it be worth storing the start offset here as well?

Reply:
I was wondering about it. I think the start offset used elsewhere is more meant for indicating where a palette starts in a larger section of data. I'm not sure it would add value here.

Reply:
We might revisit this. I'm thinking Palette might go more the direction of Surface, allowing the data to be passed in with a smaller size, specifically around cursors.

Reply:
I'll mark this resolved as I feel it falls out of scope for the pull request.